jdaviz/app.py

```diff
@@ -1673,7 +1719,24 @@ def compose_viewer_area(viewer_area_items):
 
         for name in config.get('tray', []):
             tray = tray_registry.members.get(name)
-            tray_item_instance = tray.get('cls')(app=self)
+            tray_registry_options = tray.get('viewer_reference_name_kwargs', {})
+
+            optional_tray_kwargs = dict()
+
+            if tray_registry_options is not None:
```

Comment:
Is it possible for `tray_registry_options` to be `None` (instead of an empty dict)?